root/etc/e-smith/templates/etc/pam.d/cockpit/10base

```diff
@@ -0,0 +1,24 @@
+#%PAM-1.0
+auth	   required	pam_sepermit.so
+auth       substack     password-auth
+auth       include      postlogin
+auth       optional     pam_ssh_add.so
+# the root and members of domain admins are always allowed
+# if pam auth succeed, it ignores  pam_listfile.so
+auth [success=2 default=ignore] pam_succeed_if.so user in [root]
+auth [success=1 default=ignore] pam_succeed_if.so user ingroup [domain admins]
+# select group allowed to login to cockpit (root & domain admins alway OK)
```

Comment:
I think we can remove the above lines and implement the same rule in `cockpit.allow` template logic.

Reply:
well if something goes wron in the template expansion ,then root and doamin admins are locked out ....it could be dangerous

I think that root and domain admins group must be accepted all the time

Reply:
done